core-services/sanitize-prow-jobs/_config.yaml

```
@@ -1125,6 +1125,966 @@ buildFarm:
+      - openshift-priv-apiserver-library-go-release-4.22-presubmits.yaml
+      - openshift-priv-apiserver-library-go-release-4.10-presubmits.yaml
+      - openshift-priv-apiserver-library-go-release-4.15-presubmits.yaml
+      - openshift-priv-apiserver-library-go-release-4.16-presubmits.yaml
+      - openshift-priv-controller-runtime-common-release-4.22-presubmits.yaml
+      - openshift-priv-apiserver-library-go-release-4.19-presubmits.yaml
+      - openshift-priv-apiserver-library-go-release-4.20-presubmits.yaml
+      - openshift-priv-apiserver-library-go-release-4.11-presubmits.yaml
+      - openshift-priv-apiserver-library-go-release-4.14-presubmits.yaml
+      - openshift-priv-apiserver-library-go-release-4.13-presubmits.yaml
+      - redhat-openshift-ecosystem-cvp-ocp-4.12-periodics.yaml
+    build03:
```

YAMLlint (1.38.0)

[error] duplication of key "build03" in mapping

(key-duplicates)